test/src/test/java/org/corfudb/integration/WorkflowIT.java

```diff
@@ -37,35 +38,30 @@ String getConnectionString(int port) {
     final Duration pollPeriod = Duration.ofMillis(50);
     final int workflowNumRetry = 3;
 
+    private Process runServer(int port, boolean single) throws IOException {
```

Comment:
If there is also a parameter for host, this helper method would be more flexible.

Reply:
We already have a generic function as CorfuRunner. I just wanted a common function for this class.